src/Formatter/MoneyFormatter.php

```diff
@@ -4,14 +4,10 @@
 
 namespace Setono\SyliusFacebookPlugin\Formatter;
 
-final class MoneyFormatter
+final class MoneyFormatter implements MoneyFormatterInterface
```

Comment:
Maybe should we just remove this service and create a trait for this? Wdyt?

Reply:
A separate service with interface is easier to override for plugin users in order to fix/improve something.